src/htmx.js

```diff
@@ -1923,10 +1919,13 @@ return (function () {
 
         function findElementsToProcess(elt) {
             if (elt.querySelectorAll) {
-                var boostedElts = hasChanceOfBeingBoosted() ? ", a" : "";
-                var results = elt.querySelectorAll(VERB_SELECTOR + boostedElts + ", form, [type='submit'], [hx-sse], [data-hx-sse], [hx-ws]," +
+                var boostedElts = document.querySelectorAll("[hx-boost] a, [data-hx-boost] a");
```

Comment:
Shouldn't we also query for `a[hx-boost]` and `a[data-hx-boost]` there, i.e. query the links that define themselves a `hx-boost` attribute instead of only links that are children of a boosted element?